src/StaticFAQ/ArticleDetail/Article.js

```diff
@@ -60,7 +60,7 @@ type ComponentProps = {
 };
 
 type Props = ComponentProps & {
-  section: FAQSectionType,
+  section: ?FAQSectionType,
```

Comment:
is there a chance when it's not set?

Reply:
Yes. Technically it should represent the "loading" value, but I can't use that here because I don't want to change behavior in this PR. In fact, in my fixes here https://github.com/kiwicom/smart-faq/pull/654 I want to use `null` as loading value. (So that components that depend on booking state can display a loading screen)

Reply:
Basically currently in production BEFORE_BOOKING section is loaded while the booking data is loaded, and then the actual section is loaded. We'll be able to use `null` value of section to fix that in https://github.com/kiwicom/smart-faq/pull/654